apps/web/pages/api/send-daily-video-transcript.ts

```diff
@@ -0,0 +1,186 @@
+import { createHmac } from "crypto";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { z } from "zod";
+
+import { getAllTranscriptsAccessLinkFromRoomName } from "@calcom/core/videoClient";
+import { sendDailyVideoTranscriptEmails } from "@calcom/emails";
+import logger from "@calcom/lib/logger";
+import { safeStringify } from "@calcom/lib/safeStringify";
+import { defaultHandler } from "@calcom/lib/server";
+import { getTranslation } from "@calcom/lib/server/i18n";
+import prisma, { bookingMinimalSelect } from "@calcom/prisma";
+import type { CalendarEvent } from "@calcom/types/Calendar";
+
+const testRequestSchema = z.object({
+  test: z.enum(["test"]),
+});
+
+const log = logger.getSubLogger({ prefix: ["send-daily-video-transcript-handler"] });
+
+const schema = z
+  .object({
+    version: z.string(),
+    type: z.string(),
+    id: z.string(),
+    payload: z
+      .object({
+        meeting_id: z.string(),
+        end_ts: z.number().optional(),
+        room: z.string(),
+        start_ts: z.number().optional(),
+      })
+      .passthrough(),
+    event_ts: z.number().optional(),
+  })
+  .passthrough();
+
+async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (!process.env.SENDGRID_API_KEY || !process.env.SENDGRID_EMAIL) {
+    return res.status(405).json({ message: "No SendGrid API key or email" });
+  }
```

Comment:
```suggestion
```

Not needed. Sengrid keys are only used for workflow emails. This email uses SMTP settings